content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -211,9 +211,9 @@ Regular <abbr>dependencies</abbr> and all kinds of <abbr>outputs</abbr>
 (including [metrics](/doc/command-reference/metrics) and
 [plots](/doc/command-reference/plots) files) are also listed (per stage) in
 `dvc.lock`, but with an additional field to store the hash value of each file or
-directory tracked by DVC: `md5` for local file system dependencies and SSH
-<abbr>external dependencies</abbr>, `etag` for HTTP, S3, Azure, and Google
-Cloud, and `checksum` for HDFS.
+directory tracked by DVC: `md5` for local file system dependencies and SSH and
+Google Cloud <abbr>external dependencies</abbr>; `etag` for HTTP, S3, and Azure;
+and `checksum` for HDFS.
```

Comment:
"and SSH and Google Cloud"? Too many ands.

Anyway, thanks but please roll this back, it's out of scope (I already did it elsewhere). Let's try to focus pls.

Reply:
Ohhhh wait I see you started editing this file because I asked for the semicolons. Thanks! Yeah my bad, that was out of scope sorry. Please roll this back if you don't mind because I've already changed all this in #1527.